layouts/partials/tabs.html

```diff
@@ -60,9 +60,47 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
+                       <div><a href="https://seaborn.pydata.org"><img src="images/content_images/v_seaborn.png" alt="visualization figure" align="middle" style="border-radius: 10px"></a></div>
+                       <div><a href="https://demo.bokeh.org/surface3d"><img src="images/content_images/v_bokeh.png" alt="visualization figure" align="middle" style="border-radius: 10px"></a></div>
+                       <div><a href="https://napari.org"><img src="images/content_images/v_napari.png" alt="visualization figure" align="middle" style="border-radius: 10px"></a></div>
+                       <div><a href="http://vispy.org/gallery.html"><img src="images/content_images/v_vispy.png" alt="visualization figure" align="middle" style="border-radius: 10px"></a></div>
+
+                     </div> 
+                    </p>
+                  </div>
+                  <div>
+                     <p>
+                     </p>
+                     <p>
+                       <a href="https://www.slideshare.net/Visage/data-visualization-101-how-to-design-chartsandgraphs">Data Visualization</a> exposes patterns, trends and correlations in textual-data, making it easier for humans to analyse and interpret large volumes of data. 
+                     </p>
+                     <p>
+                       <a href="https://python-graph-gallery.com">Visualization elements</a> such as bar graphs, pie charts, line charts, maps, infographics, dashboards, geographic maps, heatmaps, and interactive images offer valuable insights for making data-driven decisions.
+                     </p>
+                  </div>
+                  <div>
+                     <p>
+                       NumPy is the key data transformation building block for the burgeoning <a href="https://pyviz.org/overviews/index.html">Python visualization landscape</a> comprising of <a href="https://matplotlib.org">Matplotlib</a>, <a href="https://seaborn.pydata.org">Seaborn</a>, <a href="https://plot.ly">Plotly</a>, <a href="https://altair-viz.github.io">Altair</a>, <a href="https://docs.bokeh.org/en/latest/">Bokeh</a>, <a href="https://github.com/yhat/ggpy">ggplot</a>, <a href="http://vispy.org">Vispy</a> and <a href="https://github.com/napari/napari">Napari</a>, to name a few. 
+                     </p>
+                     <p>
+                       By performing parallel operations on large arrays, all at once, NumPy accelerates data-processing and visualization of large quantities of data, beyond Python's native performance levels, for data visualization at scale.
+                     </p>
+                  </div>
+                  <div>
+                     <p>
+                       <a href="https://rougier.github.io/python-visualization-landscape/landscape-colors.png"><img src="https://rougier.github.io/python-visualization-landscape/landscape-colors.png" alt="pyviz-visualization" align="left"></a> 
```

Comment:
Can you put this image in `images/` in this repo as well and make it a relative link? That way we'll make sure the page won't be broken if the image gets moved.

Reply:
Done Ref https://github.com/numpy/numpy.org/pull/64/commits/4d193d978e593b8a18b36aeb74824d0e4303f1a7